examples/test-upgrade/cluster/old/main.tf

```diff
@@ -0,0 +1,30 @@
+resource "mongodbatlas_project" "test" {
```

Comment:
Instead of old and updated can we do version numbers?   Will be more sustainable I think over time.